qemu/tests/single_driver_install.py

```diff
@@ -122,7 +122,10 @@ def run(test, params, env):
         error_context.context("Uninstalling previous installed driver",
                               test.log.info)
         for inf_name in _pnpdrv_info(session, device_name, ["InfName"]):
-            uninst_store_cmd = "pnputil /f /d %s" % inf_name
+            pnp_cmd = "pnputil /delete-driver %s" % inf_name
+            pnp_cmd += " /uninstall /force"
+            uninst_store_cmd = params.get("uninst_store_cmd" % inf_name,
+                                          pnp_cmd)
```

Comment:
We have two string formatting here, but the cfg file has only one "%s".
It will lead to 'TypeError: not all arguments converted during string formatting'.

Reply:
Updated, thanks for figuring it out.